flow/layers/platform_view_layer.cc

```diff
@@ -23,7 +23,14 @@ void PlatformViewLayer::Preroll(PrerollContext* context,
                       "does not support embedding";
     return;
   }
-  context->view_embedder->PrerollCompositeEmbeddedView(view_id_);
+  std::unique_ptr<EmbeddedViewParams> params =
```

Comment:
this doesn't need to be a `unique_ptr`. Just make this a stack variable.

Reply:
This is a refactoring just landed in a previous PR. https://github.com/flutter/engine/pull/9640

Reply:
got it.